front/src/pages/Settings.vue

```diff
@@ -13,48 +12,54 @@
     <div class="q-pa-md">
       <div class="row q-col-gutter-md">
         <div class="col-md-6 col-xs-12">
-          <q-card v-if="me" bordered>
+          <q-card v-if="me">
             <q-form @submit="changeProfile">
               <q-card-section class="row justify-between">
                 <div class="text-h6">Change Profile</div>
-                <user-badge :profile="tmpProfile" />
+                <user-badge
+                  style="margin-top: 2px"
+                  class="q-ma-none"
+                  :profile="tmpProfile"
+                />
               </q-card-section>
 
-              <q-separator class="q-mx-xl" />
+              <q-card-section class="q-pt-none q-pb-sm q-gutter-sm">
+                <div class="row">
+                  <div class="col q-pr-sm">
+                    <q-input
+                      v-model="username"
+                      filled
+                      dense
+                      label="Display name"
+                      lazy-rules
+                      @keyup.enter="changeProfile"
+                    >
+                      <template #prepend>
+                        <q-icon name="badge" />
+                      </template>
+                    </q-input>
+                  </div>
 
-              <q-card-section class="q-gutter-lg">
-                <div class="col">
-                  <q-input
-                    v-model="username"
-                    filled
-                    label="Username"
-                    hint="Displayed name"
-                    lazy-rules
-                    :rules="[
-                      (val) =>
-                        (val && val.length > 0) || 'Please type something',
-                    ]"
-                    @keyup.enter="changeProfile"
-                  >
-                  </q-input>
-                </div>
-                <div class="col">
-                  <color-picker v-model="color" label="color" />
+                  <div class="col" style="max-width: 40px">
+                    <color-picker v-model="color" icon="palette" />
+                  </div>
```

Comment:
You may choose to ignore this but with this minimalistic design it is a little less clear that I can choose my own account color. Especially because my name with my color and this color-picker button color are displayed with the same colors. But I also like this design more, so maybe if you click on your own username it will open the color-picker anyway?

Reply:
Lets leave it like this and maybe we get some (negative) feedback, otherwise it should be fine.